lib/google_search_data_viewer/keyword/google_search_parser.ex

```diff
@@ -0,0 +1,66 @@
+defmodule GoogleSearchDataViewer.Keyword.GoogleSearchParser do
+  @css_search_selectors %{
+    top_adwords: "#tads > .uEierd a.sVXRqc",
+    top_non_adwords: ".MhgNwc a",
+    non_adwords: ".yuRUbf a",
+    bottom_adwords: "#bottomads .uEierd a.sVXRqc"
+  }
+
+  def get_url_statistics(html) do
```

Comment:
The name is quite confusing, how about naming it `parse_html_content`? 💭

Reply:
I had Thanh ask me to change it to something less generic and from memory, this was the name to be chosen.

I originally had `get_url_data_from_html`, so do you suggest changing this still?

Reply:
Changed to `parse_html_content`